compiler/mio-tflite260/src/Helper.cpp

```diff
@@ -23,22 +24,52 @@ namespace mio
 namespace tflite
 {
 
-// This will provide v3/v3a format neutral BuiltinOperator
+/**
+ * This will provide v3/v3a format neutral BuiltinOperator
+ *
+ * This function referenced
+ * https://github.com/tensorflow/tensorflow/blob/7d12007d7800d3714a02e05059f3ea602d1aec78/tensorflow/lite/schema/schema_utils.cc
+ * and
+ * https://github.com/tensorflow/tensorflow/commit/cf28969fb2ea72c9738cd3acdddc0b69fd7dff5e
+ */
 ::tflite::BuiltinOperator builtin_code_neutral(const ::tflite::OperatorCode *opcode)
 {
   assert(opcode != nullptr);
-  int8_t dp_code = opcode->deprecated_builtin_code();
-  // 127 is max of int8_t which is upper bound of v3 builtin_code
-  // NOTE TensorFlow uses 'BuiltinOperator_PLACEHOLDER_FOR_GREATER_OP_CODES' for 127
-  if (dp_code < 127 && dp_code >= 0)
-    return ::tflite::BuiltinOperator(dp_code);
-  return opcode->builtin_code();
+  return std::max(opcode->builtin_code(),
+                  static_cast<::tflite::BuiltinOperator>(opcode->deprecated_builtin_code()));
```

Comment:
please do not change. I intended to code as is.

Reply:
is existing code wrong?

Reply:
[Tensorflow guide](https://github.com/tensorflow/community/blob/2c8c1d727a5a46472b24c6adbc0fb17f376f8ae3/rfcs/20200901-tflite-builtin-code-extension.md#how-to-read-builtin_code) and [schema comment](https://github.com/tensorflow/tensorflow/blob/7d12007d7800d3714a02e05059f3ea602d1aec78/tensorflow/lite/schema/schema_v3a.fbs#L987-L989) are saying that `builtin_code()` should be considered first, not `deprecated_builtin_code()`.
And then detailed implementation is updated by https://github.com/tensorflow/tensorflow/commit/cf28969fb2ea72c9738cd3acdddc0b69fd7dff5e because of some issue.
Could you let me what was your intention? :)

---

FYI, current code return incorrect builtin code for [following example](https://github.com/tensorflow/tflite-micro/blob/25da8245417b834ced53df722d6a0f517eb425b4/tensorflow/lite/micro/test_helpers.cc#L154).
```
deprecated_builtin_code = 0
builtin_code = 128 // CUMSUM
```